setup.cfg

```diff
@@ -23,6 +23,7 @@ exclude = doc,
 
           # individual files
           yt/visualization/_mpl_imports.py,
+          yt/utilities/command_line.py,
```

Comment:
you should also remove the inline comment in that file (might need to rebase this branch :/)